source/includes/steps-auto-schema-generation.yaml

```
@@ -0,0 +1,47 @@
+title: In the :guilabel:`rules` page of the MongoDB Realm UI, click :guilabel:`Generate Schema` to automatically create a Realm Schema
+ref: generate-schema
+content: |
+
+  Clicking `Generate Schema` will automatically generate a Realm Schema based on your cluster data. If you haven't already :atlas:`insert data into your cluster
+  </tutorial/insert-data-into-your-cluster/>`. 
+
+  .. figure:: /images/generate-schema-based-on-cluster-data.png
+     :width: 750px
+     :lightbox:
+
+---
+title: Edit your ``Realm Schema``
+ref: edit-json-schema
+content: |
+
+  Optionally, edit your schema to specify additional formatting on your data. 
+
+  .. figure:: /images/edit-schema-based-on-cluster-data.png
+     :width: 750px
+     :lightbox:
+
+---
+title: In the :guilabel:`SDKs` page, view your ``Realm Schema`` as a Realm Object Model in your language of choice
+ref: view-realm-db-schema
+content: |
+
+  .. figure:: /images/translate-jsonschema-into-realm-data-model.png
+     :width: 750px
+     :lightbox:
+
+---
+title: Click :guilabel:`View all Schema Errors`
```

```suggestion
title: View and Address Schema Errors
```

- "click" shouldn't be in the title. Explain _what_, not _how_.
- Presumably I want to do more than just view them